solutions/USACO jan18-billboard.mdx

```diff
@@ -0,0 +1,66 @@
+---
+id: usaco-759
+source: USACO Bronze 2018 January
+title: Blocked Billboard II
+author: Melody Yu
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_billboard_bronze_jan18.html)
+
+<LanguageSection>
+<JavaSection>
+See official editorial: http://www.usaco.org/current/data/sol_lemonade_silver_open18.html
+</JavaSection>
+
+<PySection>
+
+## python Implementation
+
+```py
+fIn, fOut = open("billboard.in"), open("billboard.out", "w")
+
+x1, y1, x2, y2 = tuple(map(lambda x: int(x), fIn.readline().split()))
+x3, y3, x4, y4 = tuple(map(lambda x: int(x), fIn.readline().split()))
```

Comment:
You don't need to convert the map into a tuple ;)

Reply:
got it!